### ISSUE TITLE
TODO alter MenuBarLyrics for verrrry long lyrics

### ISSUE THREAD

**Issue by revintec** [MartianZ/DynamicLyrics#54]
sometimes the lyrics is very long
and there's not enough room left in the status bar
the effect is that entire lyrics is not showing
leaving an ugly blank area(no text, no icon)
always mistaken that LyricsX is crashed or misbehaved

a better way would be automatically split up lyrics according to
both the available room in status bar and the length of the lyrics
at least show a "…" mark instead of get all blanked out by OSX

**Comment by ajkblue** [comment]
I agree that this needs to be fixed. Many times I have thought that the app has crashed, and have quit and restarted many times only to find out that the lyrics were hidden because of length. It would be alright to split it, or show ... at the or something so users don't think that the app has crashed. Maybe the lyrics could scroll horizontally right to left like a marquee when this happens. But something needs to be done.
